themes/default/content/blog/mlops-the-ai-challenge-is-cloud-not-code/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 date: 2023-08-30
-title: "The Real AI Challenge is Cloud, not Code!"
+title: "The Real AI Challenge is Cloud, not the AI!"
```

Comment:
I'm not a fan of this change. I’d recommend leaving the existing title in place; it still works just fine with the revised introductory text.